crates/sim2h/src/lib.rs

```diff
@@ -65,6 +83,44 @@ impl Sim2h {
         sim2h
     }
 
+    /// recalculate arc radius for our connections
+    fn recalc_rrdht_arc_radius(&mut self) {
+        let mut peer_record_set = RValuePeerRecordSet::default()
+            // sim2h is currently omniscient
+            .arc_of_included_peer_records(Arc::new(0.into(), ARC_LENGTH_MAX));
+        for (_k, c_state) in self.connection_states.read().iter() {
+            match c_state {
+                ConnectionState::Limbo(_) => (),
+                ConnectionState::Joined(_, _, dht_data) => {
```

Comment:
Here we are neglecting the space that agent is in and assuming all agents to be in the space, right? I think we should calculate r-values per space.